build.sbt

```diff
@@ -45,6 +46,7 @@ val scalaCheckVersion = "1.13.0"
 val scalaMacrosVersion = "2.1.0"
 val scalaTestVersion = "2.2.6"
 val slf4jVersion = "1.7.21"
+val scalapbVersion = "0.5.19" // inner protobuf-java version must match beam/dataflow-sdk one
```

Comment:
keep these `val *Version` sorted? The only exception is the first `dataflowSdkVersion` or `beamVersion` to make code porting easier.